### ISSUE TITLE
how the elliptical information is injected into the model during the training phase in code?

### ISSUE THREAD

**Issue by WHUminghui** [NVlabs/protcomposer#5]
Hi!

Thank you for the meaningful work you have done. I have been reading your code line by line.

When I run the following command to sample:
`python sample.py --guidance 1.0 --num_prots 6 --nu 5 --sigma 6 --helix_frac 0.4 --seed 1 --outdir results --num_blobs 9 --ckpt model_weights/trained_on_pdb.ckpt`

In _proteinblobs/multiflow/flow_model.py_, when if self.args.blob_attention is True, the elliptical information is fed into the model, generating proteins corresponding to the ellipses.

However, when I run the following command to train:
`python train.py --batch_size 8 --designability --designability_freq 5 --num_designability_prots 50 --accumulate_grad 8 --inf_batches 5 --val_batches 5 --finetune --dataset multiflow --self_condition --num_workers 10 --save_val --epochs 1000 --run_name my_run_name --wandb
`
Here, _if self.args.blob_attention_ is _False_, and I did not find any interaction between the elliptical information and the protein data in the later stages.

Could you please tell me where in the code the elliptical information is injected into the model during the training phase?
